tasks/lzards-backup/src/index.ts

```diff
@@ -286,6 +300,8 @@ export const backupGranule = async (params: {
       file,
       collectionId,
       granuleId: granule.granuleId,
+      provider: granule.provider,
+      createdAt: granule.createdAt,
```

Comment:
Please add test to verify that the granule metadata is submitted to lzards.
https://github.com/nasa/cumulus/blob/master/example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js#L433

Reply:
The task output doesn't have provider and collectionId.
https://us-east-1.console.aws.amazon.com/states/home?region=us-east-1#/executions/details/arn:aws:states:us-east-1:596205514787:execution:jao-ci-tf-IngestAndPublishGranule:194f16e0-b9d9-4d91-8d9e-dfa922d72158
`"backupStatus": [
      {
        "statusCode": 201,
        "granuleId": "MOD09GQ.A5682897.p5Ts1X.006.0497957657364",
        "filename": "s3://cumulus-test-sandbox-public/MOD09GQ___006/MOD/jao-ci-tf-IngestGranuleSuccess-1647621775802/MOD09GQ.A5682897.p5Ts1X.006.0497957657364_ndvi.jpg",
        "createdAt": 1647621972108,
        "body": "{\n  \"id\": 31765\n}",
        "status": "COMPLETED"
      }
    ]`

Reply:
I've updated the integration test to verify the provider and createdAt I added are returned in the `backupStatus`. I can/will add collectionId as well.

Reply:
Added validation for collection as well.

Reply:
Can you use `getAuthToken` from @cumulus/lzards-api-client in the task?